go/logger/test_logger.go

```diff
@@ -17,108 +18,121 @@ import (
 // a *testing.T).  We define this in order to avoid pulling in the
 // "testing" package in exported code.
 type TestLogBackend interface {
-	Error(args ...interface{})
-	Errorf(format string, args ...interface{})
-	Fatal(args ...interface{})
-	Fatalf(format string, args ...interface{})
-	Log(args ...interface{})
 	Logf(format string, args ...interface{})
+	FailNow()
 }
 
 // TestLogger is a Logger that writes to a TestLogBackend.  All
 // messages except Fatal are printed using Logf, to avoid failing a
 // test that is trying to test an error condition.  No context tags
```

Comment:
Any reason for not logging context tags? Is it because it's too spammy for unit tests? Would it be useful for the DSL/fuse/docker tests?